README.md

```diff
@@ -166,6 +166,205 @@ This solution collects anonymous operational metrics to help AWS improve the
+	- Ex: `"sourceKey: "myTestBucket"`
+	
+- `sourceKey`: string
+	- The S3 key of the data set to be normalized in the `sourceBucket`
+	- Ex: `"sourceKey": "myTestData.json"`
+	
+- `timestampColumn`: string
+	- Column name containing timestamps for the time series data sets
+	- Can be left as an empty string if the data set is going to be a dimension type (not time series)
+	- Ex: `"timestampColumn": ""`
+	
+	- If the data set is going to be a fact type (time series), then a `timestampColumn` must be defined to be one of the data set's columns
+	- Ex: `"timestampColumn": "timestamp"`
+
+### Outputs
+- Transformed data files in the user-specified `output_bucket` partitioned according to AMC spec
+- JSON response with the `JobRunId`
+- Ex: `"JobRunId": "jr_123123123123123123aaaaaaaaaa"`
+
+## Using Postman
+
+Postman can be used to send a request to the deployed instance's `start_amc_transformation` and begin.
+
+The instance's endpoint will be the output `UserInterface` URL from the CloudFormation base stack followed by `/api/start_amc_transformation`. AWS Signature credentials are required to access the endpoint.
+
+Ex: `https://abc123.execute-api.us-east-1.amazonaws.com/api/start_amc_transformation`
+
+1. This URL can be entered in Postman for a POST method request.
+2. In a terminal, run the following command to get AWS Signature credentials to send a request to the endpoint:
+
+```shell
+aws sts get-session-token --profile $USER
+```
+
+- The `$USER` parameter should be the profile in your `aws/.credentials` file that is for the AWS account that the CloudFormation stack is deployed in
+- If a `InvalidClientTokenId` error is thrown when running the command, try getting fresh credentials from the AWS Console as the ones being used may have expired
+- More information on the `get-session-token` command can be found here:
+	- `https://docs.aws.amazon.com/cli/latest/reference/sts/get-session-token.html`
+
+The object returned should look like below if the command works:
+```json
+{
+
+    "Credentials": {
+
+        "AccessKeyId": "ABCDEFGHIJKLMNOPQRS",
+
+        "SecretAccessKey": "abc123efg",
+
+        "SessionToken": "abcdefg1234567890",
+
+        "Expiration": "2023-02-28T11:09:14+00:00"
+
+    }
+
+}
+```
+
+
+3. Under the request's `Authorization` tab, set up the AWS Signature authorization as the following:
+
+- `Type`: AWS Signature
+- `Add authorization data to`: Request Headers
+- `Access Key`: generated from command
+- `Secret Key`: generated from command
+- `AWS Region`: us-east-1
+- `Service Name`: can leave empty
+- `Session Token`: generated from command
+
+4. Under the request's `Body` tab, select a raw JSON body and enter the body for the request. Below are 4 example request bodies for different requests:
+
+Example body for uploading to a new dimension dataset:
+```JSON
+{
+	"countryCode": "US",
+	"datasetId": "test-dataset1",
+	"deletedFields": "[]",
+	"outputBucket": "myStack-etl-artifacts",
+	"period": "P1D",
```

Comment:
remove "period" argument

Reply:
For this and the others below, I took from the UI request and it actually has these in the request body -- but will remove from these examples